### ISSUE TITLE
About Future Module Documentation

### ISSUE THREAD

**Issue by sky-joker** [ansible-collections/community.zabbix#98]
##### SUMMARY

Up until now, the module documentation was generated automatically in Ansible upstream, but I think we need to create it for each Collection in the future.

I found this tool.
https://github.com/ansible-network/collection_prep

The above tool is an automation build module document for collections.
However, I don't know if this tool will be used by default in Collections.

Do you have any suggestions on what to do about the module documentation?

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME

all modules

**Comment by D3DeFi** [comment]
None :(

@gundallow sorry to bother you. Are there some additional official instructions since [this March post](https://groups.google.com/forum/#!topic/ansible-devel/oKqgCeYTs-M) that I've perhaps missed?

**Comment by D3DeFi** [comment]
cc @gundalow . I've miss-spelled it the first time, not sure if notification was sent after edit

**Comment by gundalow** [comment]
As `community.zabbix` will be included in ansible 2.10 we will build docs for it. This has taken a little longer than expected. Making sure that `ansible-test sanity` passes should be enough for us to know we can generate the docs.

**Comment by gundalow** [comment]
TL;DR Nothing for you to do, but wait a little longer

**Comment by sky-joker** [comment]
@gundalow 

Is it correct to recognize that the Zabbix module documentation will be built with Sphinx as before by Ansible team?

**Comment by sky-joker** [comment]
The Zabbix modules documentation is now available on the following page.

http://docs.testing.ansible.com/ansible/2.10/collections/community/zabbix/index.html#plugins-in-community-zabbix
